lms/envs/production.py

```diff
@@ -1048,3 +1048,8 @@ def get_env_setting(setting):
 CHROME_DISABLE_SUBFRAME_DIALOG_SUPPRESSION_TOKEN = ENV_TOKENS.get(
     'CHROME_DISABLE_SUBFRAME_DIALOG_SUPPRESSION_TOKEN', CHROME_DISABLE_SUBFRAME_DIALOG_SUPPRESSION_TOKEN
 )
+
+################# Bulk Course Email Settings #################
+BULK_COURSE_EMAIL_LAST_LOGIN_ELIGIBILITY_PERIOD = ENV_TOKENS.get(
```

Comment:
I believe these are only needed for old variables. See https://github.com/edx/edx-platform/blob/master/lms/envs/production.py#L65-L90 for more details.